usegalaxy.org/assembly.yml.lock

```diff
@@ -316,7 +316,7 @@ tools:
 - name: pretext_graph
   owner: iuc
   revisions:
-  - 75b4a2298714
+  - eca7d3a0c5f2
```

Comment:
The role doesn't uninstall anything, so keep the other revision listed. We want that for reproducibility anyway.
```suggestion
  - 75b4a2298714
  - eca7d3a0c5f2
```